src/test/coins_tests.cpp

```diff
@@ -108,7 +108,7 @@ class CCoinsViewCacheTest : public CCoinsViewCache
 
 BOOST_FIXTURE_TEST_SUITE(coins_tests, BasicTestingSetup)
 
-static const unsigned int NUM_SIMULATION_ITERATIONS = 40000;
+static const unsigned int NUM_SIMULATION_ITERATIONS = 5000;
```

Comment:
This test is really important and central to finding any issues with the coins cache.  I know it takes a long time to run...probably the most time consuming by far of all the tests, especially in debug... For my bike-shedding two cents I would prefer to see it at least at 10000...which is still a big savings.

Reply:
I bumped it, its no big issue. I am just trying to get travis to not time out as much.  

Here's an idea: Since this PR adds the ability to have command line options into test_bitcoin, we could have a short and long option and pass --short to travis.